### DISCUSSION TITLE
Arduino_APDS9960

### DISCUSSION THREAD

**Discussion by atghrr** [SpenceKonde/megaTinyCore#707] [Library discussion]
Hello!
I'm trying to use default Arduino library to work with apds9960, but a compilation error occurs. Apds9960 uses I2C interface. The chip is Attiny3227.
Previously, when i used atmega168(328) it was workable.
Please help...

There is a log of Arduino IDE:
C:\Users\1\Documents\Arduino\libraries\Arduino_APDS9960\src\Arduino_APDS9960.cpp: In member function 'size_t APDS9960::readBlock(uint8_t, uint8_t*, unsigned int)':
C:\Users\1\Documents\Arduino\libraries\Arduino_APDS9960\src\Arduino_APDS9960.cpp:261:41: error: call of overloaded 'requestFrom(int, unsigned int&)' is ambiguous
     _wire.requestFrom(APDS9960_ADDR, len);
                                         ^
In file included from C:\Users\1\Documents\Arduino\libraries\Arduino_APDS9960\src/Arduino_APDS9960.h:24:0,
                 from C:\Users\1\Documents\Arduino\libraries\Arduino_APDS9960\src\Arduino_APDS9960.cpp:20:
C:\Users\1\Documents\ArduinoData\packages\megaTinyCore\hardware\megaavr\2.5.11\libraries\Wire\src/Wire.h:106:13: note: candidate: uint8_t TwoWire::requestFrom(uint8_t, uint8_t)
     uint8_t requestFrom(uint8_t address, uint8_t quantity);
             ^~~~~~~~~~~
C:\Users\1\Documents\ArduinoData\packages\megaTinyCore\hardware\megaavr\2.5.11\libraries\Wire\src/Wire.h:108:13: note: candidate: uint8_t TwoWire::requestFrom(uint8_t, size_t)
     uint8_t requestFrom(uint8_t address, size_t  quantity);
             ^~~~~~~~~~~
C:\Users\1\Documents\ArduinoData\packages\megaTinyCore\hardware\megaavr\2.5.11\libraries\Wire\src/Wire.h:110:13: note: candidate: uint8_t TwoWire::requestFrom(int, int)
     uint8_t requestFrom(int     address, int     quantity);
             ^~~~~~~~~~~
exit status 1

**Comment by SpenceKonde** [comment]
Converted thread to issue, locking thread. please continue discussion in #708
